site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -20,74 +20,42 @@
  **********************************************************************************/
 package org.sakaiproject.site.tool;
 
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
We've generally been avoiding .* imports in Sakai because it does slightly slow down the compilation and it can cause class-name conflicts that arise in later versions of these libraries. But it looks like it's used in 291/5878 (5%) of java files in Sakai core already.